src/GitHub.UI.Reactive/Controls/Validation/UserErrorMessages.cs

```diff
@@ -46,7 +46,7 @@ public UserErrorMessages()
             };
         }
 
-        public static readonly DependencyProperty IconMarginProperty = DependencyProperty.Register("IconMargin", typeof(Thickness), typeof(UserErrorMessages), new PropertyMetadata(new Thickness(0,10,7,0)));
+        public static readonly DependencyProperty IconMarginProperty = DependencyProperty.Register("IconMargin", typeof(Thickness), typeof(UserErrorMessages), new PropertyMetadata(new Thickness(0,0,8,0)));
```

Comment:
Hmmm, I wonder if the error there is meant to appear... we do have some errors that show up inline.